### PR TITLE
[#4231] Clarify that only the main command triggers events

### PR DESCRIPTION
**Pull request by riperez**
| Q | A |
| --- | --- |
| Doc fix? | yes |
| New docs? | no |
| Applies to | all |
| Fixed tickets | #4231 |

### CODE REVIEW

**Comment by riperez** [comment]
This is my first contribution so please be kind to me ;)

File: components/console/events.rst [hunk 1]
```diff
@@ -20,6 +20,11 @@ the wheel, it uses the Symfony EventDispatcher component to do the work::
     $application->setDispatcher($dispatcher);
     $application->run();
 
+.. caution::
+
+    Console application lifecycle events are only triggered by the main command being executed. Inner or nested
```
Comment (WouterJ) [inline components/console/events.rst]: I think we should call them "Console events" (just like we did in the versionadded directive).

File: components/console/events.rst [hunk 2]
```diff
@@ -20,6 +20,11 @@ the wheel, it uses the Symfony EventDispatcher component to do the work::
...
+    commands being called from the main command will not be triggering any of the following events.
```
Comment (WouterJ) [inline components/console/events.rst]: What about "Commands called by the main command will not trigger any event."?

**Comment by WouterJ** [comment]
It's a very good start! Correct branch, fixing a ticket, using correct syntax, etc. Great!

Besides my two pretty minor comments above, you should add a line break after the first word that crosses the 72th character. This does include the 4 spaces of indent.

**Comment by riperez** [comment]
Second version following @WouterJ advice. Thanks for the help!

**Comment by WouterJ** [comment]
:+1: from me

**Comment by xabbuh** [comment]
@riperez Looks great to me too. :+1:

**Comment by weaverryan** [comment]
Looks perfect to me! Thanks Richard for the PR. Always even better to have a PR that closes an issue too :). Cheers!

**Pull request merged**